File: Selenium/Activities/Activity3.py

from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net")

    print(driver.title)

    driver.find_element(By.XPATH, "//a[text() = 'About Us']").click()

    print(driver.title)

    driver.quit()
    
with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net/selenium/login-form")

    print(driver.title)

    driver.find_element(By.XPATH, "//*[@id='username']").send_keys("admin")

    driver.find_element(By.XPATH, "//*[@id='password']").send_keys("password")

    driver.find_element(By.XPATH, "//button[text()='Log in']").click()

    print(driver.title)

    driver.quit()

File: TestNG/Activities/Activity3.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity3 {

	
	WebDriver driver;

	@BeforeClass
	public void setUp() {

		driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/login-form");

	}
	
	@Test
	public void test5() {
		
		driver.findElement(By.id("username")).sendKeys("admin");
		driver.findElement(By.id("password")).sendKeys("password");
		driver.findElement(By.xpath("//*[text()='Log in']")).click();
		String conformation =driver.findElement(By.id("action-confirmation")).getText();
		Assert.assertEquals(conformation, "Welcome Back, admin");
	}
	
	@AfterClass
	public void teardown() {
		driver.quit();
	}

}
